lib/numo/sparse/csc.rb

```diff
@@ -49,6 +50,25 @@ def nnz
         @indices = Numo::Int32[*indices]
         @indptr = Numo::Int32[*indptr]
       end
+
+      def to_narray()
+        matrix = data.class.zeros(shape)
+        col, current, curr_ind = 0, 0, 0
+        curr_data, col_lim, curr_ptr = 0, 0, 1
+        while col < (indptr.size - 1)
+          col_lim = (indptr[curr_ptr] - indptr[curr_ptr-1])
+          while current < col_lim
+            matrix[indices[curr_ind]][col] = data[curr_data]
+            curr_ind += 1
+            curr_data += 1
+            current += 1
+          end
+          curr_ptr += 1
+          col += 1
+          current = 0
+        end
+        matrix
+      end
```

Comment:
@david81982 Could you please remove the change in this file because the COO matrix is not the target of this pull-request.